templates/init/redhat/elasticsearch.j2

```diff
@@ -67,7 +67,7 @@ if [ ! -z "$CONF_FILE" ]; then
 fi
 
 exec="$ES_HOME/bin/elasticsearch"
-prog="{{es_instance_name}}_{{default_file | basename}}"
+prog="{{es_instance_name + '_' if es_use_shared_fs else ''}}{{default_file | basename}}"
```

Comment:
Would it be possible to move all of `'_' if es_use_shared_fs else ''` logic into `defaults/main.yml`? It makes it a lot more readable and means that any future changes won't need to keep copy pasting this if else statement everywhere.